document-store/src/main/java/org/hypertrace/core/documentstore/model/subdoc/UpdateOperator.java

```diff
@@ -6,4 +6,5 @@ public enum UpdateOperator {
   REMOVE_ALL_FROM_LIST,
   ADD_TO_LIST_IF_ABSENT,
   APPEND_TO_LIST,
+  INCREMENT
```

Comment:
Still think if we name this as `ADD`, 
1. it can be extended to adding the values of 2 fields and stored in a 3rd one
2. pairity operators like `MULTIPLY` can be introduced in the future